packages/block-library/src/quote/block.json

```diff
@@ -3,12 +3,9 @@
 	"name": "core/quote",
 	"category": "text",
 	"attributes": {
-		"value": {
-			"type": "string",
-			"source": "html",
-			"selector": "blockquote",
-			"multiline": "p",
-			"default": ""
+		"withCitation": {
```

Comment:
It's a bit odd to have a persisted attribute to control citation visibility, can you explain a bit what's the thinking?

Reply:
Yeah, this seems like something that belongs in local state (via `useState`). Also, it should probably be called `hasCitation`.

Reply:
Yeah, we don't need this. https://github.com/WordPress/gutenberg/pull/25892/commits/f4aac64183c4d58e4ab26e56de819d39d1c05ae3